src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
about to push an update with the annotation version ... I'll let that clarify

and thanks your clarification helped - i think i still like the annotation approach better, but take a look when i push and we'll go from there